internal/tests.js

```diff
@@ -119,6 +119,9 @@ function mochaTestTask() {
                 return;
             }
             process.exit(1);
+        }) // https://github.com/sindresorhus/gulp-mocha#test-suite-not-exiting
+        .once('end', function () {
```

Comment:
I honestly don't know whether `on` or `once` is more appropriate here. @tfennelly ?